routes.js

```diff
@@ -73,79 +73,83 @@ module.exports = async function (app, opts) {
     app.setErrorHandler(defaultErrorHandler)
   }
 
-  app.get('/graphql', {
+  app.post('/graphql', {
```

Comment:
why this was moved from get to post? It make this very hard to review

Reply:
I thought it is better to have the `post` first as it is always the same. But you're right, it is really hard to review so changed the order back.